### DISCUSSION TITLE
Welcome to Match3-SDK Discussions!

### DISCUSSION THREAD

**Discussion by ChebanovDD** [ChebanovDD/MatchSweets#2] [Announcements]
## 👋 Welcome!
Welcome to Match3-SDK Discussions! This is a place to connect with other members of our community.

We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded

To get started, comment below with an introduction of yourself or ask any questions you’re wondering about.

**Comment by restush** [comment]
@ChebanovDD thank you for making open source for Match 3.
